PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+# Pull Requests Guidelines
+
+See [CONTRIBUTING.md](./CONTRIBUTING.md) for more details about when to create
+a GitHub [Pull Request][1] and when other kinds of contributions or
+consultation might be more desirable.
+
+When creating a new pull request, please fork the repo and work within a
+development branch.
+
+## Commit Messages
+
+* Most changes should be accompanied by tests.
+* Commit messages should explain _why_ the changes were made.
+```
+Summary of change in 50 characters or less
+
+Background on why the change is being made with additional detail on
+consequences of the changes elsewhere in the code or to the general
+functionality of the library. Multiple paragraphs may be used, but
+please keep lines to 72 characters or less.
+```
+
+## Reviews
+
+* Perform a self-review.
+* Make sure the Travis CI build passes.
+* Assign a reviewer once both the above have been completed.
+
+## Merging
+
+* Once a CEL maintaner has approved the change, it may be merged.
```

Comment:
I would change this to `After a CEL maintainer has approved the change, it may be merged by the author if the author has write access. Otherwise it will be merged by a maintainer.`

Reply:
Sounds good. I've made a minor edit to say 'If a CEL maintainer' rather than 'After' or 'Once' as there is no guarantee of approval, but kept the rest of the suggested phrasing.